app/features-json/jwt_auth.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "../services/dot_keys_variable_service"
+
+require "jwt"
+
+module FastlaneCI
+  # API Middleware responsible of authenticate all the requests that uses it.
+  class JwtAuth
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      options = { algorithm: "HS256", iss: "fastlane.ci" }
```

Comment:
@minuscorp could you expand on what led you to use HS256 instead of RS256. I'm not familiar with the benefits of one over the other. I have heard that RS256 is better. So i'm interested in known what your thoughts are here.

Reply:
So this was not a completely thought, I saw that in closed-systems was safe to use `HS256` (the provider and the consumer of the JWT are the same source, both know the encryption key): https://stackoverflow.com/a/39239395/4161167 If we're thinking on opening the application and have several encryption keys depending on the `iss` (that's another topic for discussion)